### PR TITLE
Add Cloud Alchemist haiku

### PR DESCRIPTION
**Pull request by marcosbaesse**

### CODE REVIEW

File: _haikus/Cloud_Alchemist.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Cloud Alchemist
+author: marcosbaesse
+---
+
+I'm a Cloud Alchemist<br>
```
Comment (ltagliaferri) [inline _haikus/Cloud_Alchemist.md]: This is one syllable too many, maybe you can get rid of "I'm"?

```
A Cloud Alchemist<br>
```

File: _haikus/Cloud_Alchemist.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+I've found the Philosopher's Stone<br>
```
Comment (ltagliaferri) [inline _haikus/Cloud_Alchemist.md]: This is also one syllable too long, you can try:

```
Found the Philosopher's Stone<br>
```

File: _haikus/Cloud_Alchemist.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+But now, it's a Philosopher's Service<br>
```
Comment (ltagliaferri) [inline _haikus/Cloud_Alchemist.md]: This line needs to be 5 syllables only, maybe something like...

```
But now, a Service<br>
```

**Comment by marcosbaesse** [comment]
> Thank you for submitting this clever poem! There are some changes that need to be made in order to format this into a haiku correctly. Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

Thank you for the tips. I did some changes. I haven't read that part of contribution guidelines before, sorry.

**Pull request merged**